srcpkgs/ruby-addressable/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'ruby-addressable'
+pkgname=ruby-addressable
+version=2.5.2
+revision=1
+noarch=yes
+build_style=gem
+depends="ruby-public_suffix>=2.0.2 ruby-public_suffix<4.0"
```

Comment:
I thought we don't want to evaluate ~> anymore?

Reply:
the template was generated before that changed